lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,234 @@
-import { FormattedList, FormattedMessage } from 'react-intl'
+import { Bell, BellSlash, Calendar, Clock } from '@styled-icons/fa-regular'
+import { FormattedMessage, useIntl } from 'react-intl'
 import React from 'react'
 
 import { dayFieldsToArray } from '../../../util/monitored-trip'
 import { MonitoredTripProps } from '../types'
-import FormattedDayOfWeek from '../../util/formatted-day-of-week'
-import Strong from '../../util/strong-text'
+import LocationIcon from '@opentripplanner/location-icon'
+import styled from 'styled-components'
 
-import TripSummary from './trip-summary'
+import { InlineLoading } from '../../narrative/loading'
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+
+import MonitoredDays from './trip-monitored-days'
+import TripSummary from './trip-duration-summary'
 
 /**
  * Displays the summary information of a monitored trip.
  */
+
+const SavedTripBody = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 0 0 0 15px;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+    padding: 0;
+  }
+`
+
+const LocationDetails = styled.div`
+  align-items: start;
+  border-right: 1px solid #ddd;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  padding: 20px;
+  padding-right: 30px;
+  width: 50%;
+
+  @media (max-width: 768px) {
+    align-items: center;
+    border-bottom: 1px solid #ddd;
+    border-right: 0;
+    border-top: 1px solid #ddd;
+    padding: 30px 20px 30px 20px;
+    width: 100%;
+  }
+`
+const ItineraryDetails = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+  padding-left: 30px;
+
+  @media (max-width: 768px) {
+    padding: 30px 20px 20px 20px;
+    width: 100%;
+  }
+`
+const TextWIcon = styled.div`
+  align-items: flex-start;
+  display: flex;
+  gap: 7px;
+  justify-content: left;
+  width: 250px;
+
+  svg {
+    flex-shrink: 0;
+  }
+`
+
+const TripDetailWithIcon = styled(TextWIcon)`
```

Comment:
This feels like we're reinventing something we already wrote elsewhere (and this one has some bugs). No way to re-use the renderers we use inside itinerary body?

Reply:
Are you talking about `LegDescription`? That has some really specific set line-heights and min-heights so I don't think it'd be easy to reuse. What are the bugs with this one?

Reply:
What component does `LegDescription` wrap?

Reply:
Is this what you're talkign about? 
![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/6d087e37-eef2-40fe-9969-13cb36cb8ee1)

Reply:
No!

Reply:
`route-block.tsx`

Reply:
Wait, this `TripDetailWithIcon` component are these: 
![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/600fc46d-5a64-42db-beb2-70cdb3252257)
And the route blocks are these: 
![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/62aab530-40d2-4e20-a029-2678cf871249)
And we are using the route blocks from `route-block.tsx`! It's in the `RouteBlockWrapper` in `saved-trip-list.tsx`

Reply:
Sorry I think I got myself confused again. Whatever the case though we should not be assembling the route blocks here. We already have code that does that

Reply:
Let me know if you'd like to schedule a call to figure this out

Reply:
Sure, I'll put something on the cal!